general/base_import/import_faq.rst

```diff
@@ -0,0 +1,201 @@
+=======================
+Import CSV file to Odoo
+=======================
+
+
+
+Frequently Asked Questions
+--------------------------
```

Comment:
why is this empty? if you mean the points below are the questions, just remove it